docs/design-system/sizing.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Design System/Size and Spacing" />
+
+# Size and Spacing
+
+Baklava uses a list of defined size variables.Using the right sizes will create consistency and hierarchy in user interface.
+
+| Variable                |      Value      |   <div style={{ width: '25rem' }}>Example</div>  |
+|-----------------------|:-------------:|:----------:|
+| --bl-size-4xs |  0.125rem (2px)   | <div style={{ width: '5rem', height: '0.125rem', backgroundColor: '#d9e0e8' }}></div> |
```

Comment:
Can we use variables here for setting Example height? Like `height: 'var(--bl-size-4xs)'`

Reply:
It would be great to generate this table programmatically. So we could be sure it'll always udpated and no repeating works... Maybe in the future 😊

Reply:
We'll be able to do it once we release the bl-data-grid maybe ? 🤞